src/mlpack/methods/perceptron/perceptron_main.cpp

```diff
@@ -139,9 +152,11 @@ static void mlpackMain()
 
   // If the user isn't going to save the output model or any predictions, we
   // should issue a warning.
-  RequireAtLeastOnePassed({ "output_model", "output" }, false,
+  RequireAtLeastOnePassed({ "output_model", "output", "predictions" }, false,
       "no output will be saved");
+  // "output" can be removed in mlpack 4
   ReportIgnoredParam({{ "test", false }}, "output");
```

Comment:
Same here, I think we can remove this line.

Reply:
Removed